otherlibs/unix/unix.mli

```diff
@@ -543,6 +543,28 @@ val map_file :
   validation fails.
   @since 4.06.0 *)
 
+val flush_mapped_file :
+  ('a, 'b, 'c) CamlinternalBigarray.genarray -> bool -> unit
+(** Synchronize a memory-mapped file with its in-memory view.
+   Consider a bigarray [b] obtained by {!Unix.map_file} with the [shared]
+   flag set to true.  [flush_mapped_file b sync] causes changes made
+   to the bigarray [b] to be performed on the file as well.  If [sync] is
+   true, [flush_mapped_file] blocks until the file has been fully updated.
+   If [sync] is false, the file update is scheduled but [flush_mapped_file]
+   returns immediately.
+
+   Until [flush_mapped_file] is called, the data modified in [b] can
+   stay in memory and not update the file for arbitrary long amounts
+   of time.  In particular, [flush_mapped_file] must be called before
+   the underlying file descriptor initially passed to {!Unix.map_file}
+   is closed; otherwise, depending on the OS and file system, there is
+   a risk that the changes made to [b] are lost.
+
```

Comment:
> and not update the file for arbitrary long amounts of time

When `sync = false`, I assume there is no guarantee about how long it could take for the synchronization to complete, right?  Is is still guarantee that close the file descriptor immediately after calling `flush_mapped_filed` is safe?